vnext/Desktop/DevSupportManager.cpp

```diff
@@ -121,28 +110,19 @@ task<void> DevSupportManager::LaunchDevToolsAsync(const string &debugHost, const
   auto t = task_from_result();
 
   return t.then([=]() {
-    Url url(DevServerHelper::get_LaunchDevToolsCommandUrl(debugHost));
-    auto const resolveResult = m_resolver.resolve(url.host, url.port);
-    tcp::socket socket{m_context};
-    boost::system::error_code ec;
-    connect(socket, resolveResult, ec);
-    if (ec) {
-      m_exceptionCaught = true;
-      return;
-    }
-
-    request<string_body> request{verb::get, url.Target(), 11};
-    request.set(field::host, url.host);
-    request.set(field::user_agent, BOOST_BEAST_VERSION_STRING);
+    Uri uri(Common::Unicode::Utf8ToUtf16(DevServerHelper::get_LaunchDevToolsCommandUrl(debugHost)));
+    HttpClient client;
+    client.DefaultRequestHeaders().Host(HostName(uri.Host().c_str()));
+    HttpResponseMessage response = client.GetAsync(uri, HttpCompletionOption::ResponseHeadersRead).get();
```

Comment:
>get(); [](start = 99, length = 6)

It would be nice to block the thread here as well since the API is asynchronous anyway.